xline/src/storage/kv_store.rs

```diff
@@ -376,6 +376,13 @@ where
         Ok((kvs, total))
     }
 
+    /// Get previous `KeyValue` of a `KeyValue`
+    pub(crate) fn get_prev_kv(&self, kv: &KeyValue) -> Option<KeyValue> {
```

Comment:
`get_prev_kv` is invoked in `KvWatcher`. Both `kv_store` and `kvwatcher` are belonged to `storage`. So, I think `pub(super)` is enough

Reply:
Fixed